stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -103,8 +103,7 @@ internal abstract class BaseAddCardFragment(
         saveCardCheckbox = viewBinding.saveCardCheckbox
         addCardHeader = viewBinding.addCardHeader
 
-        billingAddressView.level = sheetViewModel.config?.billingAddressCollection
-            ?: PaymentSheet.BillingAddressCollectionLevel.Automatic
+        billingAddressView.level = BillingAddressView.BillingAddressCollectionLevel.Automatic
```

Comment:
we could probably just remove this line now that level already defaults to `Automatic`

Reply:
It triggers the configure function call I think

Reply:
hmm `configureForLevel` is already called in [BillingAddressView.init](https://github.com/stripe/stripe-android/blob/76e3fad333b66f8ee34717d6d5872fad85daeffd/stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt#L159), here if we set level to `Automatic`, the delegates will end up having `oldLevel == newLevel` and not trigger `configureForLevel` call

Reply:
oh good catch.  I will remove that